src/Core/Entities/Organization.cs

```diff
@@ -78,6 +78,7 @@ public class Organization : ITableObject<Guid>, ISubscriber, IStorable, IStorabl
     public int? MaxAutoscaleSmSeats { get; set; }
     public int? MaxAutoscaleSmServiceAccounts { get; set; }
     public bool SecretsManagerBeta { get; set; }
+    public bool LimitCollectionCdOwnerAdmin { get; set; }
```

Comment:
Might be nice to add a comment saying `Cd` means `Create/Delete`.

Reply:
Added some context - thanks!